src/Components/Components/src/Sections/HeadContent.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Components.Rendering;
+
+namespace Microsoft.AspNetCore.Components.Sections
+{
+    /// <summary>
+    /// Provides content to <see cref="HeadSection"/> components.
+    /// </summary>
+    public class HeadContent : ComponentBase
```

Comment:
This type (along with `HeadSection`) arguably fits better in `Components.Web`, but I've put it in `Components` to avoid adding the sections types to the public API or requiring an IVT from `Components` to `Components.Web`. Bringing this up in case anyone thinks it's a problem.

Reply:
I would personally be ok with IVT between Components and Components.Web. But we've in the past avoided this (e.g. with the `ElementReferenceContext`. So 🤷🏽

Reply:
I've been doing all kinds of unnatural things today to work around the absence of .Components->.Web IVT.

At this point I'd be super interested in adding that IVT. Not as a precedent for adding them everywhere, mind - just because these two packages in particular are *really* close friends, and we're wasting a bunch of energy on pretending their relationship is merely platonic.

Reply:
> and we're wasting a bunch of energy on pretending their relationship is merely platonic.

Whoah! This is a PG rated repo.